README.md

```diff
@@ -42,7 +34,7 @@ At the DPGA, we envision a growing community of contributors and stakeholders ar
 
 ## 🙏 Acknowledgements
 
-Thank you to all the individuals that contributed to and [endorsed](endorsement.md) the DPG Standard, as well as well as to the participants of the **Early Grade Reading Community of Practice**. We also thank the following organizations: UNICEF,iSPIRT, Government of Sierra Leone, Government of Norway, and the DPGA Secretariat.
+Thank you to all the individuals who contributed to the DPG Standard, as well as to the participants of the **Early Grade Reading Community of Practice**. We also thank the following organizations: UNICEF,iSPIRT, the Government of Sierra Leone, the Government of Norway, and the DPGA Secretariat.
```

Comment:
(nitpick): there should be a space before iSPIRIT.

Reply:
Good catch